tests/idbstore.c

```diff
@@ -5,29 +5,35 @@
  * found in the LICENSE file.
  */
 
+#include <stdbool.h>
 #include <stdio.h>
 #include <string.h>
 #include <assert.h>
 #include <stdlib.h>
 #include <stdio.h>
 
-#include <emscripten.h>
+#include <emscripten/emscripten.h>
+#include <emscripten/html5.h>
 
 #define DB "THE_DB"
 
 long expected;
 int result;
 
+void doExit(void* userData) {
+  emscripten_force_exit(0);
+}
+
 void ok(void* arg)
 {
   assert(expected == (long)arg);
-  REPORT_RESULT(result);
+  emscripten_set_timeout(doExit, 0, 0);
 }
 
 void onerror(void* arg)
 {
   assert(expected == (long)arg);
-  REPORT_RESULT(999);
+  assert(false);
```

Comment:
Would it be better to force_exit with a different value here? assert false will simply hang.

Reply:
`assert(false);` should never hand and always abort the entire program no better where its called from.   If its not doing that then I think that is bug, and we should fix it rather then work around it.

Are you seeing this not result in the program exiting with non-zero status?  i.e. What makes you think it will hang?

Reply:
I haven't debugged anything recently in browser tests, so maybe I'm wrong then... indeed, testing now I do see that we do not hang. Great!